Makefile

```diff
@@ -460,7 +460,7 @@ PACKAGEMAKER ?= /Developer/Applications/Utilities/PackageMaker.app/Contents/MacO
 PKGDIR=out/dist-osx
 
 release-only:
-	@if `grep -q REPLACEME doc/api/*.md`; then \
+	@if [ "$(DISTTYPE)" = "release" ] && [ `grep -q REPLACEME doc/api/*.md` ]; then \
```

Comment:
I think it might be better to check that DISTTYPE is not nightly or next-nightly.  If it's set to "custom", you presumably still want this check.

(Also, opting out is arguably better than opting in when it comes to quality checks.)